packages/widgetbook/lib/src/navigation/ui/navigation_panel.dart

```diff
@@ -52,6 +53,15 @@ class _NavigationPanelState extends State<NavigationPanel> {
           const SizedBox(
             height: 16,
           ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              ExpanderRow(
+                organizers: widget.categories,
+                size: 30,
+              ),
```

Comment:
Is there a reason to pick 30 as the size? By the way i always pick values which are dividable by 4 (or sometimes 2) because its more material design like. See this: https://material.io/design/layout/understanding-layout.html#material-measurements

Reply:
Oh I didn't know about that, I'll change it to 32 then 👍.